packages/components/src/components/logos/CoinLogo/coins.ts

```diff
@@ -15,6 +15,7 @@ export const COINS = {
     nmc: require('../../../images/coins/nmc.svg'),
     tada: require('../../../images/coins/tada.svg'),
     trop: require('../../../images/coins/trop.svg'),
+    tgor: require('../../../images/coins/trop.svg'),
```

Comment:
I have used the same logo that we have for Ropsten, but not sure if we should have any different one.

Asked for it in the issue https://github.com/trezor/trezor-suite/issues/6441#issuecomment-1283570786

Reply:
good for now. could be done anytime later.